Week2/ramazanInce.js

```diff
@@ -0,0 +1,158 @@
+console.log(`
+Question 1`);
+// 1. Write a console.log statement saying
+// "Hello World!" for each language that you know.
+
+console.log(`Hello World
+Merhaba Dunya
+Priviyet Mir
+Marhaban Dunya
+Hallo Wereld
+`);
+
+console.log(`
+Question 2`);
+// 2. Consider the following code:
+
+console.log("I'm awesome");
+
+console.log(`
+Question 3`);
+// 3. Declare a variable x and initialize it with
+// an integer, using these exact steps:
+// 3.1 First, declare your variable x (do not initialize it yet).
+// 3.2 Add a console.log statement that explains in
+// words what you think the value of x is, like in this example:
+
+var x;
+console.log('the value of my variable x will be : whateverYouThinkItWillLog');
+console.log(x);
+x = 41;
+console.log('the value of x is my age');
+console.log(x);
+
+console.log(`
+Question 4`);
+// 4. Declare a variable y and assign a string to it.
+var y = `Leersum`;
```

Comment:
You can use `let` here instead of `var` 🙂

Reply:
I did not remember but for some conseles let did not work. That is why I used var.